tools/scylla-nodetool.cc

```diff
@@ -1979,7 +1979,7 @@ void toppartitions_operation(scylla_rest_client& client, const bpo::variables_ma
     if (!table_filters.empty()) {
         params.emplace("table_filters", table_filters);
     }
-    auto res = client.get("/storage_service/toppartitions", std::move(params));
+    auto res = client.get("/storage_service/toppartitions/", std::move(params));
```

Comment:
This is quite surprising. Is this a bug in the ScyllaDB REST API?

Reply:
same here. i was surprised as well. cassandra nodetool uses `/storage_service/toppartitions`, see https://github.com/scylladb/scylla-jmx/blob/3257897ae8ed05d778038abb8a94b88c56c5be9b/src/main/java/org/apache/cassandra/service/StorageService.java#L1887 . while the definition in the json file puts `/storage_service/toppartitions/`. see https://github.com/scylladb/scylladb/blob/616eec2214a2c9446e8c18a2ad6cb54c6328dbd9/api/api-doc/storage_service.json#L108

Reply:
personally, from the RESTful API design's perspective, i'd prefer using URLs like `/users` instead of `/users/` . the latter is for a special case where the user name is empty. but probably we can fix it later. if we want to do so. what do you think?

Reply:
Yes, I think we can defer this to a follow-up. Once we retire the legacy nodetool, we can do fixes like this in  a single commit, not having to coordinate changes accross 3 repos.

Reply:
created #17578 to track it.